src/server/routes/backend-wallet/signMessage.ts

```diff
@@ -7,6 +8,7 @@ import { walletAuthSchema } from "../../schemas/wallet";
 
 const BodySchema = Type.Object({
   message: Type.String(),
+  asBytes: Type.Optional(Type.Boolean()),
```

Comment:
this maybe just me, `asBytes` makes me think, the signMessage is going to sign as bytes. maybe we can change it to `isBytes` which is clear to me, stating that the message is in bytes.

Reply:
I was thinking sign string as bytes, but can change to isBytes.